src/pcp2json/pcp2json.py

```diff
@@ -116,6 +118,10 @@ def __init__(self):
         self.space_scale_force = None
         self.time_scale = None
         self.time_scale_force = None
+        self.timeout = TIMEOUT
+        self.url = None
+        self.http_user = None
+        self.http_pass = None
```

Comment:
These should be listed few lines below in the tool specific block.

Reply:
Fixed, thanks.